src/main/java/org/morehouse/app/resources/TodosResource.java

```diff
@@ -129,4 +129,13 @@ public Response deleteTodo(@PathParam("id") int id) {
         return Response.ok().entity("row " + id + " deleted").build();
     }
 	
+	static boolean validateUpdate(Todo existing, Todo update) {
+		//return true;
+		if (existing.getAssignee() == null && update.getAssignee() == null)
```

Comment:
the fact that you are checking update.getAssignee() in both this conditional and in the else suggests that you might be able to collapse these.

i believe the following line is functionally equivalent to what you have here:
`return existing.getAssignee() == null || update.getAssignee() != null.`